index.html

```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
+            <div class="title2">Multi-Post Stories</div>
+            <div class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+        </div>
+        <ul class="items lang">
+            <li>html</li>
+            <li>css</li>
+            <li>php</li>
+            <li>mysql</li>
+            <li>javascript</li>
+            
+        </ul>
+        <div class="items buttontypeA">
+        <a href="#">See Project</a>
+        </div>
+        <div class="items site1">
+            <div class="description2">
+                <div class="title3">Multi-Post Stories</div>
+                <div class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+            </div>
+            <ul class="items lang">
+                <li>html</li>
+                <li>css</li>
+                <li>php</li>
+                <li>mysql</li>
+                <li>javascript</li>
+                
+            </ul>
+            <div class="items buttontypeB">
+                <a href="#">See Project</a>
+                </div>
+        </div><!--site1 end-->
+        <br> <br> <br>
+        <div class="items site1 site2">
+            <div class="description2">
+                <div class="title3">Multi-Post Stories</div>
+                <div class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+            </div>
+            <ul class="items lang">
+                <li>html</li>
+                <li>css</li>
+                <li>php</li>
+                <li>mysql</li>
+                <li>javascript</li>
+                
+            </ul>
+            <div class="items buttontypeB">
+                <a href="#">See Project</a>
+                </div>
+        </div><!--site2 end-->
+        
+    </div><!--container end-->
+</div><!--page2 end-->
+<br> <br> <br>
+<div class="page3">
+    <div class="container">
+        <br> <br> 
+        <div class="items title1">About me</div>
+        <div class="items text1">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </div>
+        <div class="items buttontypeA">
+            <a href="#">Get My Resume</a>
+        </div>
+        <div class="items graphic">
+            <img src="images/graphic.png" alt="">
+        </div>
+        <div class="items hr">
+            <hr>
+        </div>
+        <div class="items languages">
```

Comment:
- [ ] Kindly change this `<div>` container into a `<ul>` for all "Languages", "Frameworks", and "Skills" and use the proper `<li>` tag for the correct elements. As a result, you will have a nested `<ul>` (an `<ul>` inside another `<ul>`).